### PR TITLE
switch to behance/hogan.js@v3.0.6 with the inheritance fix

### PR DESCRIPTION
**Pull request by nemtsov**
We can finally say "goodbye" to `assets-require/hogan`.

### CODE REVIEW

**Comment by matt-oconnell** [comment]
LGTM

File: package.json [hunk 1]
```diff
@@ -32,7 +32,7 @@
     "nodemon": "^1.11.0"
   },
   "dependencies": {
-    "assets-require": "github:behance/assets-require#v1.1.0",
+    "hogan.js": "github:behance/hogan.js.git#1be31eb",
     "recursive-readdir": "^2.1.1"
   },
   "engines": {
```
Comment (agaripian) [inline package.json]: Why a commit hash?

Comment (nemtsov) [inline package.json]: Good catch. Switched to version.

**Comment by mikesherov** [comment]
👍

**Pull request merged**